.github/workflows/js-api.yml

```diff
@@ -373,13 +375,29 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         if: needs.version.outputs.changed_version == 'true'
+      - name: compatibility-tool
```

Comment:
Name it a bit more descriptive
'Commit js-api version json for compatibility-tool', for example